clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/cache/RedisCacheConfig.groovy

```diff
@@ -47,7 +47,7 @@ class RedisCacheConfig {
     JedisSource jedisSource,
     ObjectMapper objectMapper,
     @Value('${redis.maxMsetSize:250000}') int maxMsetSize) {
-    new RedisNamedCacheFactory(jedisSource, objectMapper, maxMsetSize)
+    new RedisNamedCacheFactory(jedisSource, objectMapper, maxMsetSize, null)
```

Comment:
Ya, that's what I figured. It would be nice to Spectator wired in ... doesn't have to block us from moving this forward but I'd like to see cache metrics on our dashboards sooner than later. Reasonable?

Seems like these metrics would be much more valuable then what we have right now around how often caching agents run.